src/glyph/mod.rs

```diff
@@ -86,9 +86,6 @@ impl Glyph {
         path: &Path,
         opts: &WriteOptions,
     ) -> Result<(), GlifWriteError> {
-        if self.format != GlifVersion::V2 {
-            return Err(GlifWriteError::Downgrade);
```

Comment:
Actually, why remove the check here? I'd leave it in and change the Glyph element serializer to write out any minor version other than zero. Otherwise we may silently do something the user did not see coming.

Reply:
The `Downgrade` error could also be renamed to `UnsupportedGlifVersion` because a user could also request 2.123 or something.

Reply:
We remove the check because we no longer store a version (we're always 2.0)